file_parser.py

```diff
@@ -0,0 +1,135 @@
+from collections import Counter
+from typing import List, Tuple
```

Comment:
Since *Python 3.9* [PEP 585](https://peps.python.org/pep-0585/) it is recommended to use generic standard types to where possible - **list** instead of **typing.List**, **tuple** instead of **typing.Tuple**